packages/graphql/src/builder.ts

```diff
@@ -46,7 +46,7 @@ const builder = new SchemaBuilder<{
   }
 }>({
   authScopes: async (ctx) => ({
-    authenticated: !!ctx.user?.sub,
+    authenticated: !!ctx.user?.sub && ctx.user.scope !== UserLoginScope.OTP,
```

Comment:
this should prevent usage of a magic link for login (it should only be usable to generate a real login token). has not been verified that it works like that yet